pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/NonPersistentTopics.java

```diff
@@ -374,6 +375,27 @@ public void getListFromBundle(
         });
     }
 
+    @DELETE
+    @Path("/{tenant}/{namespace}/{topic}/truncate")
+    @ApiOperation(value = "Truncate a topic.",
+            notes = "NonPersistentTopic is not support truncate.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 412, message = "NonPersistentTopic is not support truncate")
```

Comment:
NonPersistentTopic **does** not support truncate

Reply:
Thank you very much, I will fix it as soon as possible.